app/lua/lrotable.h

```diff
@@ -7,39 +7,46 @@
 #include "luaconf.h"
 #include "lobject.h"
 #include "llimits.h"
+#include "lrotable.h"
```

Comment:
Including `lrotable.h` in `lrotable.h`? Seems a bit too pious ;)

Reply:
Sorry, `lrodefs.h` had got so slim that there was no point in keeping it since it did a dozen defines then included `lrotable.h`.  Combining the two actually shortened `lrotable.h`.  So one of my bulk edits was to replace any references  `lrodefs.h` by  `lrotable.h` and remove the duplicate where we ended up with a double occurrence.   I forgot this self-reference case. :astonished: